Assets/AdditionalScripts/EndScreen/PlayAgainButtonScript.cs

```diff
@@ -8,8 +8,11 @@ public class PlayAgainButtonScript : MonoBehaviour
     public void LoadGame(string sceneName)
```

Comment:
```suggestion
    public void LoadGame()
```
remove useless parameter

Reply:
true, ty